electron/legendary/games.ts

```diff
@@ -275,25 +275,33 @@ class LegendaryGame extends Game {
   /**
    * Adds a desktop shortcut to $HOME/Desktop and to /usr/share/applications
    * so that the game can be opened from the start menu and the desktop folder.
-   * Both can be disabled with addDesktopShortcuts and addStartMenuShortcuts
    * @async
    * @public
    */
-  public async addShortcuts(fromMenu?: boolean) {
+  public async addShortcuts({addToStartMenu, addToDesktop}: {addToStartMenu: boolean, addToDesktop: boolean}) {
     if (process.platform === 'darwin') {
       return
     }
 
     const gameInfo = await this.getGameInfo()
     const launchWithProtocol = `heroic://launch/${gameInfo.app_name}`
-    const [ desktopFile, menuFile ] = this.shortcutFiles(gameInfo.title)
-    const { addDesktopShortcuts, addStartMenuShortcuts } = await GlobalConfig.get().getSettings()
+    const sanitizedTitle = gameInfo.title
+    // TODO: use regexp
+    sanitizedTitle.replaceAll('/', ' ')
```

Comment:
Doing like this will be too slow. Try using this website to find a regexp pattern to remove special character. I think we are using regex in some places across the code.
https://regexr.com/

Reply:
I don't know Regexp. Can you suggest a pattern?

Reply:
I think /:|\/|\*|\?|<|>|\|/g should work

Reply:
I think /:|/|*|?|<|>||/g should work

Reply:
`new RegExp('[:|/|*|?|<|>||]')` is working.